server/api/googleLogin.js

```diff
@@ -98,9 +83,8 @@ async function insertUser(db, username, callback) {
       }
 
       // create a user object, handle it elsewhere
-      // TODO: Perhaps this should also generate a user token at this time?
       user = {
-        user_id: this.lastID,
+        user_id: this ? this.lastID : row.lastID,
```

Comment:
note: this is needed for testing. I couldn't figure out how to specify a `this` object via the test file

Reply:
Are we doing anything with the user object? If not, it can probably be removed

Reply:
it's used to set create the session, so we need it in this one